### PR TITLE
Option to remove rabbitmq from a deployment.

### PR DESCRIPTION
**Pull request by tanviraumi**
The PR enables removal of rabbitmq from a deployed environment.

### CODE REVIEW

**Review by DLehenbauer — APPROVED** [review]

**Comment by tanviraumi** [comment]
Thanks @DLehenbauer

File: server/routerlicious/kubernetes/routerlicious/values.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ mongodb:
   url: mongodb_url
 
 rabbitmq:
-  connectionString: "connection_string"
+  connectionString: ""
```
Comment (appym31) [inline server/routerlicious/kubernetes/routerlicious/values.yaml:57]: For my own understanding, why do we make the change here and not in server/routerlicious/packages/routerlicious/config/config.json ?

Comment (tanviraumi) [reply server/routerlicious/kubernetes/routerlicious/values.yaml:57]: Cause we want to keep running rabbitmq locally. This file is what actually goes to cluster. There is another step of overriding this value but thats a deployment level change.

File: server/routerlicious/packages/services-core/src/index.ts [hunk 2]
```diff
@@ -11,6 +11,7 @@ export * from "./combinedLambda";
 export * from "./combinedProducer";
 export * from "./database";
 export * from "./document";
+export * from "./emptyTaskMessageSender";
```
Comment (appym31) [inline server/routerlicious/packages/services-core/src/index.ts:14]: Curious what this file is for and why we need to add these exports here?

Comment (tanviraumi) [reply server/routerlicious/packages/services-core/src/index.ts:14]: This is the index.js file which is an entrypoint. Adding exports here let us do things like import { Something } from "packagename"

**Pull request merged**